tests/gatsby-starter-default/test.robot

```diff
@@ -31,3 +31,10 @@ Scenario: Link back to home
   Page should not contain  gatsby-source-plone
   Click link  Learn more »
   Page should contain  gatsby-source-plone
+  Click link  gatsby-source-plone
+  Wait until page contains  Gatsby is a blazing fast
```

Comment:
@ajayns Remove these extra lines. They are merge artifacts from some earlier version and are not relly part of this test.

Reply:
On it!, But it seems my test failed as well. I need to figure out what went wrong there.